composer.json

```diff
@@ -5,7 +5,7 @@
     "type": "magento2-theme",
     "require": {
       "magento/framework": "*",
-      "snowdog/module-alpaca-components": "~0.8.0",
+      "snowdog/module-alpaca-components": "dev-feature/42921",
```

Comment:
Please change it to newer components version after release

Reply:
ok